src/components/PageHeader/PageHeader.tsx

```diff
@@ -52,17 +53,19 @@ export const PageHeader = ({
   align,
   className,
   description,
+  headingSize = 'headline-lg',
   kicker,
+  right,
   title,
   titleAfter,
-  right,
-  headingSize = 'headline-lg',
+  variant,
   ...other
 }: Props) => {
   const componentClassName = clsx(
     styles['page-header'],
     className,
     align === 'center' && styles['page-header--center'],
+    variant === '50-50' && styles['page-header--50-50'],
```

Comment:
Change this to `orientation` to leave `variant` open for more stylistic color changes that may be needed

Reply:
Changed to `orientation`, thanks!